spec/enumerables_spec.rb

```diff
@@ -0,0 +1,55 @@
+RSpec.describe Enumerable do
+  describe 'my_each' do
+    it 'returns an array with the items + 1' do
+      expect([1, 2, 3].my_each{ |item| item + 1 }).to eql([1, 2, 3])
```

Comment:
Don't repeat yourself with providing arrays for each of them, instead consider using `let`.